src/test/java/uk/ac/ebi/eva/pipeline/jobs/GenotypedVcfJobWorkflowTest.java

```diff
@@ -236,11 +232,11 @@ private JobParameters initVariantConfigurationJob() throws IOException {
                 .vepCacheSpecies("")
                 .vepCacheVersion("")
                 .vepNumForks("")
-                .vepPath(getResource(MOCK_VEP).getPath());
+                .vepPath(getResource(MOCK_VEP).getPath())
+                .annotationSkip(false)
+                .statisticsSkip(false);
```

Comment:
These 2 should default to `false`. Do we want to keep them explicit to ensure this behavior even if the main code changes?